DNN Platform/Library/Services/Installer/Installers/ScriptInstaller.cs

```diff
@@ -187,6 +202,19 @@ public override void Install()
                 {
                     bSuccess = this.InstallScriptFile(this.UpgradeScript);
                     installedVersion = this.UpgradeScript.Version;
+                }

```

Comment:
Following the [documentation ](https://dnndocs.com/content/tutorials/extensions/dnn-manifest-schema/index.html#the-dnn-manifest-schema)to maintain expectations, I think it would make sense to do things in this order.

* Install
* Schema
* Upgrade

Thoughts?

Reply:
In general, I'd agree and we need to make sure to be backwards compatible under all conditions. 
Besdes see my previous comment.

Reply:
You're assuming that the upgrade script makes no changes to the schema and only to data. This is fair enough but will probably need to be documented somewhere.

Reply:
BTW, there is no issue of backwards compatibility regarding the run order and what is in documentation. This is a new feature. So backwards compatible means no schema scripts are there.